packages/donor/src/utils/StorybookMocks.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  Donor,
+  BloodType,
+  DonorNotificationSettings,
+} from "@zm-blood-components/common";
+
+const notificationSettingsMock: DonorNotificationSettings = {
+  disableEmailNotifications: true,
+};
+export const mockUser: Donor = {
```

Comment:
Nice!
Please move the file to `src/__test__`.

Also, I suggest calling the object `sampleUser` because we already have similar thing in `functions`